Week3/homework/step2-2.js

```diff
@@ -3,18 +3,28 @@
 function threeFive(startIndex, stopIndex, threeCallback, fiveCallback) {
   const numbers = [];
 
-  // Replace this comment and the next line with your code
-  console.log(startIndex, stopIndex, threeCallback, fiveCallback, numbers);
+  for (let i = 0; i <= stopIndex - startIndex; i++) {
+    numbers.push(startIndex + i);
+  }
+
+  for (const number of numbers) {
+    if (!(number % 3)) {
+      threeCallback(number);
+      if (!(number % 5)) {
+        fiveCallback(number);
+      }
+    } else if (!(number % 5)) {
+      fiveCallback(number);
+    }
+  }
 }
 
 function sayThree(number) {
-  // Replace this comment and the next line with your code
-  console.log(number);
+  console.log(`Number: ${number} is divisible by 3!`);
```

Comment:
Nice use of ES6 template literals to produce user-friendly output. Use exclamation marks sparingly though, no need to shout!